### DISCUSSION TITLE
Prevent any addiction

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#1107] [Completed Ideas] [answered]
Hi dev i have a idea to at least reduce the addiction to reddit 

1. Disable the popular and similar tab while keep the subscribed tab remain
2. Disabling comment and vote counts

Please consider this idea thanks fellow dev
🙏😊

**Comment by Docile-Alligator** [comment, marked as answer]
Settings->Interface->Customize Tabs in Main Page and delete extra tabs.

Settings->Interface->Post->Hide the Number of Votes & Hide the Number of Comments
Settings->Interface->Post Details->Hide the Number of Votes & Hide the Number of Comments
